llama.cpp/server/public/completion.js

```diff
@@ -196,7 +206,7 @@ export const llamaComplete = async (params, controller, callback) => {
 export const llamaModelInfo = async (config = {}) => {
   if (!generation_settings) {
     const url_prefix = config.url_prefix || "";
-    const props = await fetch(`${url_prefix}/props`).then(r => r.json());
+    const props = await fetch(buildUrl(url_prefix, 'props')).then(r => r.json());
```

Comment:
I just updated this one mechanically, but couldn't figure out how to test it. Upon further inspection, I can't find any references to `llamaModelInfo` in the repo. Maybe it should just be deleted?